pkg/storage/tpr/fake_core_rest_client_test.go

```diff
@@ -0,0 +1,357 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tpr
+
+import (
+	"bytes"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+
+	"github.com/gorilla/mux"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/conversion"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+
+	"k8s.io/client-go/pkg/api"
+	fakerestclient "k8s.io/client-go/rest/fake"
+
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/testapi"
+
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1alpha1"
+)
+
+type objStorage map[string]runtime.Object
+type typedStorage map[string]objStorage
+type namespacedStorage map[string]typedStorage
+
+func (s namespacedStorage) set(ns, tipe, name string, obj runtime.Object) {
```

Comment:
I must say it took me a moment to figure out what was going on here since I was trying to find a struct for namespacedStorage. Just not used to this, no need for action.